src/service/action-impl.js

```diff
@@ -402,6 +402,17 @@ export class ActionService {
     }
   }
 
+  /**
+   * Checks if the given element has registered a particular action type.
```

Comment:
Hi @choumx and @jridgewell because this exceeds the size cap. =( 

Proposing to expose a `hasAction` function from `ActionService` because `hasTapAction` is used at 3 different places in our codebase now with a regex. The proposal is to refactor `amp-story`, `amp-lightbox-gallery` to use this function. (Provided that this is a reasonable implementation...)

Reply:
LGTM.